### PR TITLE
Tradução do Attr

### PR DESCRIPTION
**Pull request by GustavoPHP**
Ola, eu traduzi o método .attr() e .detach();;

Teria como conversarmos por e-mail luiz tiago?

### CODE REVIEW

**Comment by ruanltbg** [comment]
Hem @GustavoPHP da uma olhada no seu commit, acho que vc deve ter trocado os arquivos, vc esta substituindo o arquivo do attr pelo do detach.

**Comment by ruanltbg** [comment]
ping @GustavoPHP

**Comment by ruanltbg** [comment]
Fechado por falta de atividade. A tradução não esta no arquivo certo.
